fuzzing/extract_text_fuzzer.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import atheris
+
+from fuzz_helpers import EnhancedFuzzedDataProvider
+
+with atheris.instrument_imports():
+    from pdf_utils import PDFValidator, prepare_pdfminer_fuzzing
+    from pdfminer.high_level import extract_text
+
+from pdfminer.psparser import PSException
+
+
+def TestOneInput(data: bytes):
+    if not PDFValidator.is_valid_byte_stream(data):
+        # Not worth continuing with this test case
+        return -1
+
+    fdp = EnhancedFuzzedDataProvider(data)
+
+    try:
+        with fdp.ConsumeMemoryFile() as f:
+            max_pages = fdp.ConsumeIntInRange(0, 1000)
+            extract_text(
+                f,
+                maxpages=max_pages,
+                page_numbers=fdp.ConsumeIntList(fdp.ConsumeIntInRange(0, max_pages), 2),
```

Comment:
The `page_numbers` can also be `None`. I'm wondering if we can test that as well. 

Also in the other fuzzers.